packages/sqlite-storage/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.build.json",
```

Comment:
We should use a common baseline config.

Reply:
we do, `tsconfig.build.json` extends `<root>/config/tsconfig.base.json`

Reply:
I've made it so `tsconfig.json` points to root one as well (instead of local `*.build.json`)